includes/widgets/image-carousel.php

```diff
@@ -930,27 +941,25 @@ protected function render() {
 		}
 
 		$slides_count = count( $settings['carousel'] );
-
 		?>
 		<div <?php $this->print_render_attribute_string( 'carousel-wrapper' ); ?>>
 			<div <?php $this->print_render_attribute_string( 'carousel' ); ?>>
 				<?php // PHPCS - $slides contains the slides content, all the relevent content is escaped above. ?>
 				<?php echo implode( '', $slides ); // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped ?>
 			</div>
 			<?php if ( 1 < $slides_count ) : ?>
-				<?php if ( $show_dots ) : ?>
-					<div class="swiper-pagination"></div>
-				<?php endif; ?>
```

Comment:
I have moved pagination to below the navigation arrows, so that the navigation arrows will be accessed before the pagination for users who use keyboard to navigate the page.

Reply:
@hein-obox  Love this change